storagenode/retain/retain.go

```diff
@@ -379,16 +379,16 @@ func (s *Service) retainPieces(ctx context.Context, req Request) (err error) {
 		pieceID := access.PieceID()
 		if !filter.Contains(pieceID) {
 			s.log.Debug("About to delete piece id",
-				zap.String("satellite", satelliteID.String()),
-				zap.String("pieceID", pieceID.String()),
-				zap.String("status", s.config.Status.String()))
+				zap.Stringer("Satellite ID", satelliteID),
+				zap.Stringer("Piece ID", pieceID),
+				zap.String("Status", s.config.Status.String()))
```

Comment:
nit - can this be `zap.Stringer("Status", s.config.Status)`?

Reply:
Same comment for line 411

Reply:
For some reason it can't. I was getting a compile error because I tried exactly that. Maybe next time.